es/lockwise-ios.xliff

```diff
@@ -94,23 +98,35 @@
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="10.2.1" build-num="10E1001"/>
     </header>
     <body>
+      <trans-unit id="ehm-2v-Rbb.accessibilityLabel">
+        <source>Firefox Lockwise</source>
+        <note>Class = "UIImageView"; accessibilityLabel = "Firefox Lockwise"; ObjectID = "ehm-2v-Rbb"; Note = "Text logo of the product name for screenreaders";</note>
+      </trans-unit>
       <trans-unit id="hwO-wS-oS4.text">
         <source>Take your passwords everywhere</source>
         <target>Lleva tus contraseñas a todos lados</target>
         <note>Class = "UILabel"; text = "Take your passwords everywhere"; ObjectID = "hwO-wS-oS4";</note>
       </trans-unit>
     </body>
   </file>
-  <file original="lockbox-ios/Common/Resources/InfoPlist.strings" source-language="en" target-language="es" datatype="plaintext">
+  <file original="lockbox-ios/Common/Resources/Strings/InfoPlist.strings" source-language="en" target-language="es" datatype="plaintext">
     <header>
       <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="10.2.1" build-num="10E1001"/>
     </header>
     <body>
       <trans-unit id="CFBundleDisplayName">
         <source>Lockwise</source>
-        <target>Lockwise</target>
         <note>Bundle display name</note>
       </trans-unit>
+      <trans-unit id="CFBundleName">
+        <source>Firefox Lockbox</source>
```

Comment:
With this re-inclusion it's due to the way the CFBundleName works and its tied to a bunch of things throughout the Xcode project. I tried to surgically rename it in some places but it ended up breaking things in others, sadly. I'm not clever enough to get this a) removed from the strings nor b) get it renamed in the project without breaking other assumptions elsewhere.